src/app/core/components/footer/footer.component.spec.ts

```diff
@@ -72,6 +72,8 @@ describe('Component: Footer', () => {
     const link = element.querySelector('.footer-logo a') as HTMLElement;
     expect(link).toBeTruthy();
 
+    // https://github.com/angular/angular/issues/45202
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
     router.navigate(['/about']);
```

Comment:
router.navigate returns a Promise, but is not awaited in Agora or in [the Angular docs](https://angular.io/api/router/Router#navigate). Per the referenced link, it's common practice to ignore the Promise, so disable the lint rule for these cases.